lntest/itest/assertions.go

```diff
@@ -1789,3 +1789,76 @@ func assertChannelPolicyUpdate(t *testing.T, node *lntest.HarnessNode,
 		), "error while waiting for channel update",
 	)
 }
+
+func assertTransactionNotInWallet(t *testing.T, node *lntest.HarnessNode,
+	txID chainhash.Hash) {
+
+	t.Helper()
+
+	txStr := txID.String()
+
+	err := wait.Predicate(func() bool {
+		txResp, err := node.GetTransactions(
+			context.Background(), &lnrpc.GetTransactionsRequest{},
+		)
+		if err != nil {
+			return false
+		}
+
+		notFound := true
+		for _, txn := range txResp.Transactions {
+			if txn.TxHash == txStr {
+				notFound = false
+				break
+			}
+		}
+
+		return notFound
+	}, defaultTimeout)
+	if err != nil {
+		t.Fatalf("transaction %v found in wallet", txID)
+	}
+}
+
+func assertAnchorOutputLost(t *harnessTest, node *lntest.HarnessNode,
+	chanPoint wire.OutPoint) {
+
+	pendingChansRequest := &lnrpc.PendingChannelsRequest{}
+	err := wait.Predicate(func() bool {
+		pendingChanResp, pErr := node.PendingChannels(
+			context.Background(), pendingChansRequest,
+		)
+		if pErr != nil {
+			return false
+		}
+
+		for _, pendingChan := range pendingChanResp.PendingForceClosingChannels {
+			if pendingChan.Channel.ChannelPoint == chanPoint.String() {
+				return (pendingChan.Anchor ==
+					lnrpc.PendingChannelsResponse_ForceClosedChannel_LOST)
+			}
+		}
+
+		return false
+	}, defaultTimeout)
+	if err != nil {
+		t.Fatalf("anchor doesn't show as being lost")
+	}
+}
+
+func assertNoPendingForceCloses(t *harnessTest, node *lntest.HarnessNode) {
```

Comment:
there is `assertNumPendingChannels` does this I think

Reply:
True! Swapped out for that one instead.